.github/workflows/sync-warp.yaml

```
@@ -0,0 +1,68 @@
+#
+# dotfiles
+# Sync WARP VM's dotfiles
+#
+
+name: Sync dotfiles on WARP
+on:
+  push: {}
+    #branches: [main]
+
+jobs:
+  create-warp-pr:
+    name: Create PR to update dotfiles in WARP Box
+    runs-on: ubuntu-22.04
+    env:
+      WARP_REPO: mrzzy/warp
+      COMMIT_USER: github-actions[bot]
+      COMMIT_EMAIL: github-actions[bot]@users.noreply.github.com
+      BRANCH: build/update-dotfiles
+    steps:
+      - name: Clone WARP repository
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ env.WARP_REPO }}
+          token: ${{ secrets.GH_PUSH_TOKEN }}
+
+      - name: Push updated mrzzy/dotfiles commit hash
+        run: |
+          set -ex -o pipefail
+
+          # update hash in devbox playbook to use latest dotfiles
+          DEFAULT_VARS=box/ansible/roles/devbox/defaults/main.yaml
+          git switch -c ${BRANCH}
+          sed -i -e '/devbox_dotfiles_rev:/s/[^ ]\+$/${{ github.sha }}/' ${DEFAULT_VARS}
+
+          # commit & changes to dotfiles commit hash
+          SHORT_SHA=$(printf ${{ github.sha }} | cut -c -8)
+          git config user.name ${COMMIT_USER}
+          git config user.email ${COMMIT_EMAIL}
+          git commit \
+            -m "build(ansible): bump revision of dotfiles to ${{ env.REPO }}@${SHORT_SHA}" \
```

```suggestion
            -m "build(ansible): bump revision of dotfiles to ${WARP_REPO}@${SHORT_SHA}" \
```
Typo. Also GitHub context interpolation is not necessary here as `WARP_REPO` is a env var.